app-modules/portal/resources/css/retrospective.css

```
@@ -502,6 +503,185 @@
     margin-top: 9px;
 }
 
+/* panorama do github */
+.retro .faint {
+    color: var(--faint);
+}
+.retro .pan-hero {
+    font-family: 'Fraunces', serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)